### DISCUSSION TITLE
Migrating from Laravel Mix to Vite

### DISCUSSION THREAD

**Discussion by kaloczikvn** [Laravel-Backpack/community-forum#249] [Ideas]
The latest Laravel version is using Vite by default, backpack should also use it by default. From what I experienced using Mix was really slow for bigger projects, a project of ours had a 10 minutes deploy time, with Vite we reduced it to ~1 minute.

**Comment by tabacitu** [comment]
Hey @kaloczikvn - I have even better news 😀 We're removing both Mix and Vite in Backpack v6. Hear me out! 😅

This is a pretty cool innovation we've made in the PHP ecosystem. We've written a white paper for it 3-4 years ago, thinking it should be possible to load CSS and JS assets directly from CDN, then have them internalized to the server, and used from there. That way, there would be no need to run any build tools. We didn't have time to do it, though. Then deno.land came out, proving it's actually possible and efficient, even using Javascript alone. So we've gone ahead and solved the problem in PHP too. Keep an eye on https://github.com/DigitallyHappy/assets/pull/14 for more info. We might have stumbled into something huge here.
